warehouse/postgres/postgres.go

```diff
@@ -31,16 +31,13 @@ var (
 )
 
 const (
-	host          = "host"
-	dbName        = "database"
-	user          = "user"
-	password      = "password"
-	port          = "port"
-	sslMode       = "sslMode"
-	serverCAPem   = "serverCA"
-	clientSSLCert = "clientCert"
-	clientSSLKey  = "clientKey"
```

Comment:
I was wondering if we should delete those or not. Yes, they are not used but ...

Reply:
Yeah not used + unexported. Not much use for them. Do you want to keep them as comment?